### PR TITLE
voasis

### PR DESCRIPTION
**Pull request by aquestry**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/voasis/background.png
Comment (maaaathis) [inline minecraft_servers/voasis/background.png:1]: Compress

File: minecraft_servers/voasis/background@2x.png
Comment (maaaathis) [inline minecraft_servers/voasis/background@2x.png:1]: Compress

File: minecraft_servers/voasis/manifest.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "server_name": "voasis",
+  "nice_name": "Voasis",
+  "direct_ip": "voasis.de",
+  "server_wildcards": [
+    "%.voasis.de"
```
Comment (maaaathis) [inline minecraft_servers/voasis/manifest.json:6]: Doesn't look like a real wildcard
image

File: minecraft_servers/voasis/manifest.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://voasis.de",
+    "web_support": "https://voasis.de/index.php?route=/forum",
```
Comment (maaaathis) [inline minecraft_servers/voasis/manifest.json:13]: `web_support` cannot be a forum; provide a faq or contact page or remove

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by aquestry** [comment]
i think i now have everything needed

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**